### PR TITLE
bondType -> BondFor union type

### PR DESCRIPTION
**Pull request by TingALin**
Renamed bondType to bondFor as union type

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -120,3 +120,5 @@ export enum Sync {
   Syncing,
   Synced,
 }
+
+export type BondFor = 'Pool' | 'Nominator';
```
Comment (rossbulat) [inline src/types/index.ts]: Is there a reason these strings are capitalised? Maybe just have all lower case, one less complexity.

**Review by rossbulat — CHANGES_REQUESTED** [review]
Good job, just a couple of fixes and then can merge.

File: src/types/index.ts (hunk 1 above)
Comment (TingALin) [reply src/types/index.ts]: like export type bondfor = 'pool' | 'nominator'; ?

Comment (rossbulat) [reply src/types/index.ts]: just the string literal types, not the type name.

**Review by rossbulat — APPROVED** [review]

**Pull request merged**